src/main/webapp/profileStyle.css

```diff
@@ -13,60 +13,103 @@
 // limitations under the License.
 */
 
-#content {
-  text-align: center;
+/* Ensure proper box sizing */
+* {
+  box-sizing: border-box;
+  scroll-behavior: smooth;
 }
 
-#navigation-bar {
-  margin-top: 20px;
-  height: 40px;
-  width: 600px;
-  left: 0;
-  right: 0;
-  margin-left: auto;
-  margin-right: auto;
+img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+/* Header properties */
+#header {
```

Comment:
I didn't check, but I guess these are very similar to the classed in the css file for the main page. It is usually not a good idea to have duplicated code, because you then have to remember to fix it in multiple places when you want to make a change. It is probably a good idea to move all the common stuff to a shared file (e.g. common.css) that can be referenced from all the pages. You can then just have the page-specific classes and overrides in a page-specific css file (if it is needed at all). You can either do it as part of this PR, or create an issue and add a TODO to do it in a follow-up PR.

Reply:
Done